ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -224,8 +224,18 @@ async fn get_data_with_tasks(
         Ok(CacheCoverageStatus::CacheHit(_)) => 1,
         Ok(CacheCoverageStatus::CacheEvicted) => match transactions_count {
             None => MAX_FETCH_TASKS_PER_REQUEST,
-            Some(transactions_count) => (transactions_count / TRANSACTIONS_PER_STORAGE_BLOCK)
```

Comment:
(transactions_count / TRANSACTIONS_PER_STORAGE_BLOCK) can be 

(transactions_count  + TRANSACTIONS_PER_STORAGE_BLOCK / TRANSACTIONS_PER_STORAGE_BLOCK - 1) to round up

ref: https://stackoverflow.com/questions/17944/how-to-round-up-the-result-of-integer-division

Reply:
your current proposal looks good; approved

Reply:
i refactor to if/else statement. the math by itself is too confusing.